xchange-core/src/main/java/org/knowm/xchange/service/marketdata/MarketDataService.java

```diff
@@ -108,6 +148,26 @@ default Trades getTrades(CurrencyPair currencyPair, Object... args) throws IOExc
     throw new NotYetImplementedForExchangeException("getTrades");
   }
 
+  /**
+   * Get the trades recently performed by the exchange
+   *
+   * @param args Optional arguments. Exchange-specific
+   * @return The Trades, null if some sort of error occurred. Implementers should log the error.
+   * @throws ExchangeException - Indication that the exchange reported some kind of error with the
+   *     request or response
+   * @throws NotAvailableFromExchangeException - Indication that the exchange does not support the
+   *     requested function or data
+   * @throws NotYetImplementedForExchangeException - Indication that the exchange supports the
+   *     requested function or data, but it has not yet been implemented
+   * @throws IOException - Indication that a networking error occurred while fetching JSON data
+   */
+  default Trades getTrades(Instrument instrument, Object... args) throws IOException {
```

Comment:
again same as above. lets make the `Instrument` and `CurrencyPair` methods separate so we can decommission the `CurrencyPair` methods at some point.